transport/matrix/layer.py

```diff
@@ -59,7 +42,7 @@ def __init__(self, config: Dict[str, Any]):
 
```

Comment:
it seems like there is an unresolved reference (`log`) on line `58`

Reply:
fixed